themes/openy_themes/openy_carnation/dist/img/geysir/small-banner.svg

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Should these files be on global, not on carnation only level?
somewhere like here https://github.com/ymcatwincities/openy/tree/8.x-2.x/themes/openy_themes/scss or in ```../img/``` ?